### ISSUE TITLE
PPT for Module 3

### ISSUE THREAD

**Issue by Senajs** [AzureCAT-GSI/DevCamp#136]
Pls Check PPT Slide 18  are the authentications correct ?

![image](https://cloud.githubusercontent.com/assets/15032819/21220780/80920afc-c2e0-11e6-9f35-853c9ebe1f55.png)

**Comment by nickward13** [comment]
I agree. I've created a broader issue #190 referencing this and a number of other issues in this document.
